incubator/datadog-apm/values.yaml

```diff
@@ -5,7 +5,7 @@
 nameOverride: ""
 fullnameOverride: ""
 targetSystem: linux
-namespace:  # Need to override this value with datadog
+namespace: default # Need to override this value with datadog
```

Comment:
This should be removed from the values file. We don't need to specify namespace as that's done by the helm release process.